network/rollup.config.js

```diff
@@ -0,0 +1,21 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+
+export default {
+  input: 'dist/esm/index.js',
+  output: {
+    file: 'dist/plugin.js',
+    format: 'iife',
+    name: 'capacitorPlugin', // TODO: change this
```

Comment:
We need to name plugins here according to Dans checklist. Came here to see what you called yours so I could follow the same convention :)